core/src/main/java/org/apache/accumulo/core/file/blockfile/cache/impl/BlockCacheManagerFactory.java

```diff
@@ -31,7 +31,7 @@ public class BlockCacheManagerFactory {
 
   /**
    * Get the BlockCacheFactory specified by the property 'tserver.cache.factory.class' using the
-   * AccumuloVFSClassLoader
+   * configured ContextClassLoaderFactory
```

Comment:
I think this will now use the system class loader. I don't think this is done inside a context, but I am not sure.